tests/security/usbguard/usbguard.pm

```diff
@@ -0,0 +1,138 @@
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test 'usbguard' can work:
+#          '# systemctl enable usbguard.service',
+#          '# systemctl start usbguard.service',
+#          '# systemctl status/is-active usbguard.service',
+#          '# usbguard generate-policy', '# usbguard allow-device',
+#          '# usbguard list-devices', '# usbguard block-device'
+#          '# check/cat '/var/log/usbguard/usbguard-audit.log',
+#          '# usbguard add-user', '# usbguard remove-user',
+#          '# usbguard get-parameter', '# usbguard set-parameter'
+#          '# usbguard generate/list/install/remove rules'
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#102566, tc#1769830
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $out = '';
+    my $user = 'usbguard';
+    my $allow = 'allow';
+    my $block = 'block';
+    my $msg = 'Tablet';
+    my $f_rules_default = '/etc/usbguard/rules.conf';
+    my $f_rules_test = '/etc/usbguard/rules_test.conf';
+    my $f_IPCAccessControl = '/etc/usbguard/IPCAccessControl.d/\:' . "$user";
+    my $f_usbguard_audit_log = '/var/log/usbguard/usbguard-audit.log';
+
+    select_console 'root-console';
+
+    # 0. Set up environment
+    # Install usbguard packages
```

Comment:
As I know usbguard related packages are in **Package Hub** module previous SPx , is it need to be considered if the system is without **Package Hub** module installation?

Reply:
For now for new test cases, we do not need to consider previous SPx as QEM takes this part. 
What we are suggested to consider is Factory First.
But if you like you can support/test/review/... previous SPx.

Reply:
yes, I know, I mean this case will not plan to be run in previous SPx, right?